botocore/regions.py

```diff
@@ -454,6 +455,9 @@ class EndpointResolverBuiltins(str, Enum):
     SDK_ENDPOINT = "SDK::Endpoint"
     # An account ID sourced from the credential resolution process.
     AWS_ACCOUNT_ID = "AWS::Auth::AccountId"
+    # The AWS credential scope configured for the SDK client. Clients may not
+    # make requests to regions outside their credential scope.
```

Comment:
I don't think this is quite right. Credentials are not valid outside of their credential scope region.